src/pages/components/select/style.mdx

```diff
@@ -82,13 +82,13 @@ tabs: ['Usage', 'Style', 'Code', 'Accessibility']
 
 <Tab label="Default">
 
-![Select interactive states](images/select-style-states-fixed.png)
+![Interactive states of default select](images/select-style-interactive-states-default.png)
```

Comment:
Disabled and Read-only state (fixed): You can take out the helper text for these since it is not in the other states.